aviary/modules/binning/scripts/das_tool.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+import sys
+
+import extern
+
+if __name__ == '__main__':
+    unrefined_binners_to_use = [
+        ('concoct', 'fa'),
+        ('maxbin2', 'fasta'),
+        ('vamb', 'fna')]
+    refined_binners_to_use = [
+        ('rosella', 'fna'),
+        ('semibin', 'fna')]
+
+    # N.B. specifying "metabat" will skip both MetaBAT1 and MetaBAT2.
+    metabats = ['metabat_sspec', 'metabat_ssens', 'metabat_sens', 'metabat_spec']
+
+    binners = []
+    for (binner, extension) in unrefined_binners_to_use:
+        if binner not in snakemake.config['skip_binners']:
+            binners.append((f'{binner}_bins/', extension, f'data/{binner}_bins.tsv'))
+
+    for (binner, extension) in refined_binners_to_use:
+        if binner not in snakemake.config['skip_binners']:
+            binners.append((binner+'_refined/final_bins/', extension, f'data/{binner}_refined_bins.tsv'))
+
+    for metabat in metabats:
+        if metabat not in snakemake.config['skip_binners']:
+            binners.append((metabat.replace('metabat','metabat_bins'), 'fa', f'data/{metabat}_bins.tsv'))
+    if 'metabat2' not in snakemake.config['skip_binners']:
+        binners.append(('metabat2_refined/final_bins/', 'fna', f'data/metabat2_refined_bins.tsv'))
+
+    logfile = snakemake.log[0]
+    logging.basicConfig(filename=logfile, level=logging.INFO)
+    logging.info("Using the following binners: " + str(binners))
+
+    if len(binners) == 0:
+        logging.error("All binners have been skipped, so DAS_tool cannot be run.")
+        sys.exit(1)
+    if len(binners) == 1:

```

Comment:
I'm fairly DASTool should be able to run with bins from a single binner, it would just attempt to identify contamination and remove it via the gene annotation step it uses. I haven't tested this, but I think it would run fine and we wouldn't need this check here

Reply:
Correct - I thought I'd run into that issue but something else must've been broken at the time. The test now tests for this since only semibin is used.